src/Plugin.FilePicker/Plugin.FilePicker.csproj

```diff
@@ -79,6 +83,10 @@
     <Compile Include="**\*.netstandard.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('netcoreapp3')) ">
+    <Compile Include="**\*.net45.cs" />
```

Comment:
is it correct?

Reply:
Yes,
I know it looks a bit odd, and possible a refactoring should be done somewhere in the future, but there is a parallel development for dot net core and dot net standard, and dot net framework.

.net core 3.0 most important change compared to .net core 2.x is that it added support for the WPF framework and for forms. As the .net framework version was 4.5 at the moment of releasing .net core 3.0, the WPF and Forms implementation of .net core 3.0 is almost the same as the WPF and Forms implementation of .net framework 4.5.
As a result of this , when a WPF or Form implementation is available, most of the time the .net core implementation can be done, just by using the .net framework 4.5 implementation.

I hope this explains this odd lines in the project file

Jaap

Reply:
Awesome. That totally makes sense. I'll merge it and make a beta package out of it to get some feedback on this. A BIG thank you for this wonderful contribution